app/models/katello/docker_manifest_platform.rb

```diff
@@ -0,0 +1,9 @@
+module Katello
+  class DockerManifestPlatform < Katello::Model
+    has_many :docker_manifest_manifest_platforms, :dependent => :destroy, :class_name => "Katello::DockerManifestManifestPlatform"
+    has_many :docker_manifest_platforms, :through => :docker_manifest_manifest_platforms
+
+    has_many :docker_manifest_list_manifest_platforms, :dependent => :destroy, :class_name => "Katello::DockerManifestListManifestPlatform"
+    has_many :docker_manifest_platforms, :through => :docker_manifest_list_manifest_platforms
```

Comment:
Please explain why this is helpful:

```rb
 Katello::DockerManifestPlatform.find(1).docker_manifest_platforms
=> [#<Katello::DockerManifestPlatform:0x000000000f513808 id: 1, arch: "amd64", os: "linux", created_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00, updated_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00>,
 #<Katello::DockerManifestPlatform:0x000000000f5135d8 id: 1, arch: "amd64", os: "linux", created_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00, updated_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00>,
 #<Katello::DockerManifestPlatform:0x000000000f513470 id: 1, arch: "amd64", os: "linux", created_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00, updated_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00>,
 #<Katello::DockerManifestPlatform:0x000000000f513290 id: 1, arch: "amd64", os: "linux", created_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00, updated_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00>,
 #<Katello::DockerManifestPlatform:0x000000000f512fe8 id: 1, arch: "amd64", os: "linux", created_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00, updated_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00>,
 #<Katello::DockerManifestPlatform:0x000000000f512e30 id: 1, arch: "amd64", os: "linux", created_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00, updated_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00>,
 #<Katello::DockerManifestPlatform:0x000000000f512cc8 id: 1, arch: "amd64", os: "linux", created_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00, updated_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00>,
 #<Katello::DockerManifestPlatform:0x000000000f512b10 id: 1, arch: "amd64", os: "linux", created_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00, updated_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00>,
 #<Katello::DockerManifestPlatform:0x000000000f512908 id: 1, arch: "amd64", os: "linux", created_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00, updated_at: Tue, 14 Aug 2018 13:39:04 UTC +00:00>]
```

Reply:
Definitely something wrong here as there are two _has_many :docker_manifest_platforms_

Reply:
Corrected